.github/APPLICATION_SUCCESS.md

```diff
@@ -1,4 +1,4 @@
-# Application Submitted! 🙌
+# Application Submitted! 🙌 
```

Comment:
@Fils7 avoid this kind of modification. Do `git diff` and double-check before you commit changes into the repository.

```suggestion
# Application Submitted! 🙌
```